python-challenges/automated-file-organizer-v2/devashmit/organizer.py

```
@@ -0,0 +1,54 @@
+CATEGORIES = {
+    "images":    {".jpg",".jpeg",".png",".gif",".bmp",".svg",".webp",".ico",".tiff"},
+    "documents": {".pdf",".doc",".docx",".xls",".xlsx",".ppt",".pptx",".txt",".md",".csv"},
+    "videos":    {".mp4",".mov",".avi",".mkv",".wmv",".flv",".webm"},
+    "audio":     {".mp3",".wav",".flac",".aac",".ogg",".m4a"},
+    "archives":  {".zip",".tar",".gz",".rar",".7z"},
+    "code":      {".py",".js",".ts",".html",".css",".json",".xml",".yaml",".sh",".jsx",".tsx"},
+}
+
+def get_category(ext: str) -> str:
+    for cat, exts in CATEGORIES.items():
+        if ext.lower() in exts: return cat
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)